src/hooks/useSWRContract.ts

```diff
@@ -120,6 +121,10 @@ export function useSWRContract<
   )
 }
 
+export function useSWRMulticall(abi: any[], calls: Call[], options: MulticallOptions = { requireSuccess: true }) {
+  return useSWR(calls, () => multicallv2(abi, calls, options))
```

Comment:
Use this in caution with multicall. By default, it refetch when on focus or mount.

Reply:
Noted